client/jetpack-onboarding/constants.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal Dependencies
+ */
+import React from 'react';
+import JetpackOnboardingContactFormStep from './steps/contact-form';
+import JetpackOnboardingHomepageStep from './steps/homepage';
+import JetpackOnboardingSiteTitleStep from './steps/site-title';
+import JetpackOnboardingSiteTypeStep from './steps/site-type';
+import JetpackOnboardingSummaryStep from './steps/summary';
+import JetpackOnboardingWoocommerceStep from './steps/woocommerce';
+
+export const JETPACK_ONBOARDING_STEPS = {
+	SITE_TITLE: 'site-title',
+	SITE_TYPE: 'site-type',
+	HOMEPAGE: 'homepage',
+	CONTACT_FORM: 'contact-form',
+	WOOCOMMERCE: 'woocommerce',
+	SUMMARY: 'summary',
+};
+
+export const JETPACK_ONBOARDING_COMPONENTS = {
+	[ JETPACK_ONBOARDING_STEPS.SITE_TITLE ]: <JetpackOnboardingSiteTitleStep />,
+	[ JETPACK_ONBOARDING_STEPS.SITE_TYPE ]: <JetpackOnboardingSiteTypeStep />,
+	[ JETPACK_ONBOARDING_STEPS.HOMEPAGE ]: <JetpackOnboardingHomepageStep />,
+	[ JETPACK_ONBOARDING_STEPS.CONTACT_FORM ]: <JetpackOnboardingContactFormStep />,
+	[ JETPACK_ONBOARDING_STEPS.WOOCOMMERCE ]: <JetpackOnboardingWoocommerceStep />,
+	[ JETPACK_ONBOARDING_STEPS.SUMMARY ]: <JetpackOnboardingSummaryStep />,
+};
```

Comment:
Do we need all this indirection? I'd just turn `JETPACK_ONBOARDING_COMPONENTS`' keys into string literals. If we ever need `JETPACK_ONBOARDING_STEPS` separately, we can define 'em as `Object.keys( JETPACK_ONBOARDING_COMPONENTS )`.

Reply:
I'm more inclined to leave them like that. We'll be referring to those steps and components in other places in the code, and it will generally be much safer to refer to them by using the constants.